static/css/style.css

```diff
@@ -391,3 +391,8 @@ section .column img {
         margin-top: 0;
     }
 }
+
+.new-list {
+     margin: 0 auto;
+     width: 1200px;
```

Comment:
This will lead to a responsive issue, you can check your site with a mobile resolution

Reply:
I've removed this section